### PR TITLE
1.16.0

### LINKED ISSUE #1850: Error while using "wrangler init --site" with version 1.15.x
#### Describe the bug

Error while trying to use init command

#### Reproduce the bug

A list of steps to reproduce the bug.

2. `npx @cloudflare/wrangler@1.15.1 init --site`
2. Error:
`Error: values must be emitted before tables`

#### Expected behavior

Site initiated without errors as with version 1.14.1

#### Environment and versions

Fill out the following information about your environment.

* operating system: Ubuntu 20.04
* output of `wrangler -V`:
* output of `node -v`: v14.15.1
* content of `wrangler.toml`:

### PR DESCRIPTION
**Pull request by xortive**
closes #1850 

## 1.16.0

- ### Features

  - **Custom Builds and Modules - [xortive], [caass], [pull/1855]**

    Custom Builds and Modules has finally landed in a main release!
    There's too many new features to write about in a changelog, so here's a
    [link to the docs](https://developers.cloudflare.com/workers/cli-wrangler/configuration#build).

    [xortive]: https://github.com/xortive
    [caass]: https://github.com/caass
    [pull/1855]: https://github.com/cloudflare/wrangler/pull/1855

  - **add `--format` option, including default `json` and new `pretty` - [caass], [pull/1851]**

    It may make sense to have this just be a --pretty flag, but if we decide we want to add more formats in the future (like forwarding raw devtools protocol stuff) then it's better to have the --format flag. Plus, we can introduce a warning li
    ... truncated

    [caass]: https://github.com/caass
    [pull/1851]: https://github.com/cloudflare/wrangler/pull/1851

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+    There's too many new features to write about in a changelog, so here's a
+    [link to the docs](https://developers.cloudflare.com/workers/cli-wrangler/configuration#build).
+
+    [xortive]: https://github.com/xortive
+    [caass]: https://github.com/caass
+    [pull/1855]: https://github.com/cloudflare/wrangler/pull/1855
+
+  - **add `--format` option, including default `json` and new `pretty` - [caass], [pull/1851]**
+
+    You can now pass `--format pretty` to `wrangler tail` to get pretty printed logs!
```
Comment (caass) [inline CHANGELOG.md:19]: Maybe also 
> `--format json` is also available, and is used by default.

File: src/settings/toml/manifest.rs [hunk 2]
```diff
@@ -54,7 +54,7 @@ pub struct Manifest {
     pub vars: Option<HashMap<String, String>>,
     pub text_blobs: Option<HashMap<String, PathBuf>>,
     pub triggers: Option<Triggers>,
-    #[serde(default, with = "string_empty_as_none")]
+    #[serde(with = "string_empty_as_none")]
```
Comment (caass) [inline src/settings/toml/manifest.rs:57]: Was this intentional?

Comment (xortive) [reply src/settings/toml/manifest.rs:57]: yep! this was a bug that caused #1850

Comment (Electroid) [reply src/settings/toml/manifest.rs:57]: yes, it caused a bug

File: CHANGELOG.md (hunk 1 above)
Comment (xortive) [reply CHANGELOG.md:19]: done

**Review by caass — APPROVED** [review]

**Review by Electroid — APPROVED** [review]

**Pull request merged**